DuckDuckGo/Subscription/ViewModel/SubscriptionSettingsViewModel.swift

```diff
@@ -26,40 +26,55 @@ import Subscription
 @available(iOS 15.0, *)
 final class SubscriptionSettingsViewModel: ObservableObject {
     
+    enum Constants {
+        static let autoRenewable = "Auto-Renewable"
+        static let notAutoRenewable = "Not Auto-Renewable"
+        static let monthlyProductID = "ios.subscription.1month"
+        static let yearlyProductID = "ios.subscription.1year"
+    }
```

Comment:
Just a thought, maybe we should think about moving product id definitions into a central place? Not sure how it will work out with the Alpha vs multiple mac builds.

But I think PurchaseManager already holds these. So having set of monthly and yearly ids we use, and having methods to get legible strings for that (in BSK) is a good idea to handle both platforms WDYT?

Reply:
Yes, agree. These should be moved to BSK once we're using the same App and product ID for both apps.